tools/db_crashtest.py

```diff
@@ -221,6 +221,7 @@ def is_direct_io_supported(dbname):
     "allow_concurrent_memtable_write": lambda: random.randint(0, 1),
     "column_families": 1,
     "experimental_allow_mempurge": 0,
+    "experimental_mempurge_policy": "kAlternate",
```

Comment:
I'm surprised this feature is not already enabled/running (with some probability) in the automatic crash test runs. Please enable.